tests/test_command_line.py

```diff
@@ -1156,6 +1156,41 @@ def test_generate_c_source_database_name(self):
                 self.assert_files_equal(database_c,
                                         'tests/files/c_source/' + database_c)
 
+    def test_generate_c_source_output_directory(self):
+        databases = [
+            'motohawk',
+        ]
+        
+        output_directory = 'some_dir'
+
+        for database in databases:
```

Comment:
I suggest that the loop is removed as the test tests a single file.

Reply:
Just copying another test that did the same, but sure.